src/presentation/api/controller/cron.go

```diff
@@ -90,8 +97,9 @@ func (controller *CronController) Update(c echo.Context) error {
 // @Success      200 {object} object{} "CronDeleted"
 // @Router       /v1/cron/{cronId}/ [delete]
 func (controller *CronController) Delete(c echo.Context) error {
-	requestBody := map[string]interface{}{
-		"id": c.Param("cronId"),
+	requestBody, err := apiHelper.ReadRequestBody(c)
```

Comment:
Rename the helper to ReadRequestInputData please. RequestBody is one thing, query params are another.